prime-router/src/testIntegration/kotlin/transport/RESTTransportIntegrationTests.kt

```diff
@@ -373,4 +394,19 @@ class RESTTransportIntegrationTests : TransportIntegrationTests() {
         val retryItems = mockRestTransport.send(transportType, header, reportId, null, context, actionHistory)
         assertThat(retryItems).isNotNull()
     }
+    @Test
+    fun `test flexion rest transport`() {
```

Comment:
This test fails for me, not sure if I'm missing some setup or something, but I think generally integration tests should work without any setup.

Reply:
It is fixed, now.